ppp headers for lwip 2.0.0
===================================================================
diff -uNr lwip/cortexM4/lpc43xx/lpc4337/src/arch/lpc18xx_43xx_emac.c lwip-ciaa/cortexM4/lpc43xx/lpc4337/src/arch/lpc18xx_43xx_emac.c
--- lwip/externals/lwip/cortexM4/lpc43xx/lpc4337/src/arch/lpc18xx_43xx_emac.c	2014-12-04 16:51:28.000000000 -0300
+++ lwip-ciaa/externals/lwip/cortexM4/lpc43xx/lpc4337/src/arch/lpc18xx_43xx_emac.c	2016-08-29 19:39:26.867093708 -0300
@@ -37,7 +37,19 @@
 #include "lwip/stats.h"
 #include "lwip/snmp.h"
 #include "netif/etharp.h"
+
+#include "lwip/init.h"
+#define	CIAA_LWIP_141	(1U << 24 | 4U << 16 | 1U << 8 | 0U)
+#define CIAA_LWIP_VERSION 	(LWIP_VERSION_MAJOR << 24   | \
+	LWIP_VERSION_MINOR << 16 |LWIP_VERSION_REVISION << 8 | LWIP_VERSION_RC)
+
+#if CIAA_LWIP_VERSION == CIAA_LWIP_141
 #include "netif/ppp_oe.h"
+#else
+/* code for lwip version 2.0.0 (development) */
+#include "netif/ppp/pppoe.h"
+#include "lwip/snmp.h"
+#endif /*  CIAA_LWIP_VERSION == CIAA_LWIP_141 */
 
 #include "lpc_18xx43xx_emac_config.h"
 #include "arch/lpc18xx_43xx_emac.h"
